src/components/PageLoader/index.test.tsx

```diff
@@ -13,7 +13,7 @@ describe('PageLoader', () => {
         </PageLoader>
       );
 
-      const spinner = container.querySelector('.animate-spin svg');
+      const spinner = container.querySelector('svg.animate-spin');
```

Comment:
How about using a test id instead of this selector? 🤔

Reply:
I am not sure if it is a common practice in Javascript to include the code for testing purposes in the business domain code; let me know your thought on this 🙇‍♂️. So I tried to avoid using test id and only use it as a last resort if it is impossible to identify the element. 

For this case, I think it is solid enough to identify the loader element by querying the SVG with a specific class (`animate-spin`) and scope it under the `container`.

What do you think? 💭

Reply:
Previously I'm not so sure about Jest but on Cypress, it is recommended to [use the test ids in their best practice](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements). I think their point is to not use the selector that might be changed later. But that's from the UI test point of view 🤔 
Then after reading [this](https://infinum.com/handbook/frontend/react/testing-best-practices#basic-component-test). I think on Jest as a unit test, maybe we better try to use selectors that the user can see and if it doesn't make sense we can use the test id, what do you think? 💭

Reply:
I checked the articles that you mentioned. It makes sense to me 👍 since there is no proper semantic selector for the loader, let's use the test id 🤩

Reply:
Added the test id in [95c8bb2](https://github.com/hanam1ni/react-surveyor/pull/60/commits/95c8bb2aab315c475ed8440499f07b711892a192)